frontend/src/metabase/query_builder/hooks/use-notebook-screen-size.ts

```diff
@@ -0,0 +1,25 @@
+import { useWindowSize } from "react-use";
+
+const INITIAL_WINDOW_WIDTH = Infinity;
+
+const BREAKPOINT = 1280;
+
+const NOT_MOUNTED_YET = undefined;
+
+type NotMountedYet = typeof NOT_MOUNTED_YET;
+
+type NotebookScreenSize = {
+  isSmallScreen: boolean | NotMountedYet;
+  isLargeScreen: boolean | NotMountedYet;
+};
+
+export const useNotebookScreenSize = (): NotebookScreenSize => {
```

Comment:
wdyt about returning a single enum, e.g. `undefined | "small" | "large"`? I find it confusing when there is a difference between `undefined` and `false` for a boolean field

Reply:
Good point :+1: 
d350c9adcde03f25ef265e4213ed182a627e583c